modules/claims_api/spec/requests/v2/veterans/rswag_disability_compensation_request_spec.rb

```diff
@@ -602,4 +603,114 @@ def make_request(example)
+
+        **Standard and fully developed claims (FDCs)**
+
+        [Fully developed claims (FDCs)](https://www.va.gov/disability/how-to-file-claim/evidence-needed/fully-developed-claims/)
+        are claims certified by the submitter to include all information needed for processing. These claims process faster#{' '}
+        than claims submitted through the standard claim process. If a claim is certified for the FDC, but is missing needed information,#{' '}
+        it will be processed as a standard claim.
+
+        To certify a claim for the FDC process, set the claimProcessType to FDC_PROGRAM.
+      VERBIAGE
+      description get_schema_description
+
+      parameter name: 'veteranId',
+                in: :path,
+                required: true,
+                type: :string,
+                example: '1012667145V762142',
+                description: 'ID of Veteran'
+
+      let(:veteranId) { '1013062086V794840' } # rubocop:disable RSpec/VariableName
+      let(:Authorization) { 'Bearer token' }
+
+      let(:scopes) { %w[system/526.override] }
+
+      parameter name: :disability_comp_request, in: :body,
+                schema: SwaggerSharedComponents::V2.body_examples[:sync_disability_compensation][:schema]
+
+      # WE MIGHT NOT NEED BOTH EXAMPLES THIS IS WIP SO JUST HAD THEM IN HERE
+      # EXPECTING THESE DOCS WOULD BE THE SAME AS THE 526, BUT WE MAY GO TO ONE
+      parameter in: :body, examples: {
+        'Minimum Required Attributes' => {
+          value: JSON.parse(Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
+                                            'disability_compensation', 'valid_526_minimum_synchronous.json').read)
+        },
+        'Maximum Attributes' => {
+          value: JSON.parse(Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
+                                            'disability_compensation', 'form_526_json_api_synchronous.json').read)
+
+        }
+      }
+
+      describe 'Getting a successful response' do
+        response '202', 'Successful response' do
+          let(:claim_date) { (Time.zone.today - 1.day).to_s }
+          let(:anticipated_separation_date) { 2.days.from_now.strftime('%Y-%m-%d') }
+          let(:data) do
+            temp = Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
+                                   'disability_compensation', 'form_526_json_api_synchronous.json').read
+            temp = JSON.parse(temp)
+            attributes = temp['data']['attributes']
+            attributes['serviceInformation']['federalActivation']['anticipatedSeparationDate'] =
+              anticipated_separation_date
+            temp['data']['attributes'] = attributes
+            temp.to_json
+            temp
+          end
+
+          let(:disability_comp_request) do
+            data
+          end
+
+          schema SwaggerSharedComponents::V2.schemas[:sync_disability_compensation]
```

Comment:
we would probably have to do something similar to [this comment](https://github.com/department-of-veterans-affairs/vets-api/pull/16853/files#r1612017968) here though..... where we set a variable or something and merge in the claimId for the schema to reflect accordingly?